esi_leap/objects/offer.py

```diff
@@ -63,9 +65,14 @@ def get_availabilities(self):
             return []
 
         conflicts = self.dbapi.offer_get_conflict_times(self)
-
+        now = datetime.datetime.now()
```

Comment:
I think after this, I'd have

```
 start_time = self.start_time if self.start_time >= now else now
```
Then I'd use start_time in various places below.